### ISSUE TITLE
NoSuchMethodError: com.intellij.lang.parser.GeneratedParserUtilBase.enter_section_

### ISSUE THREAD

**Issue by anehing** [KronicDeth/intellij-elixir#402]
# Version

4.2.0

# Exception

## Message

com.intellij.lang.parser.GeneratedParserUtilBase.enter_section_(Lcom/intellij/lang/PsiBuilder;II)Lcom/intellij/lang/PsiBuilder$Marker;

## Stacktrace

```
java.lang.NoSuchMethodError: com.intellij.lang.parser.GeneratedParserUtilBase.enter_section_(Lcom/intellij/lang/PsiBuilder;II)Lcom/intellij/lang/PsiBuilder$Marker;
	at org.elixir_lang.parser.ElixirParser.expression(ElixirParser.java:1985)
	at org.elixir_lang.parser.ElixirParser.expressionList(ElixirParser.java:1999)
	at org.elixir_lang.parser.ElixirParser.elixirFile_1_0(ElixirParser.java:1872)
	at org.elixir_lang.parser.ElixirParser.elixirFile_1(ElixirParser.java:1863)
	at org.elixir_lang.parser.ElixirParser.elixirFile(ElixirParser.java:1848)
	at org.elixir_lang.parser.ElixirParser.parse_root_(ElixirParser.java:472)
	at org.elixir_lang.parser.ElixirParser.parseLight(ElixirParser.java:466)
	at org.elixir_lang.parser.ElixirParser.parse(ElixirParser.java:19)
	at com.intellij.psi.tree.ILazyParseableElementType.doParseContents(ILazyParseableElementType.java:64)
	at com.intellij.psi.tree.IFileElementType.parseContents(IFileElementType.java:43)
	at com.intellij.psi.impl.source.tree.LazyParseableElement.ensureParsed(LazyParseableElement.java:172)
	at com.intellij.psi.impl.source.tree.LazyParseableElement.getFirstChildNode(LazyParseableElement.java:212)
	at com.intellij.psi.impl.source.tree.LazyParseableElement.getFirstChildNode(LazyParseableElement.java:36)
	at com.intellij.psi.impl.source.tree.SharedImplUtil.getFirstChild(SharedImplUtil.java:52)
	at com.intellij.psi.impl.source.PsiFileImpl.getFirstChild(PsiFileImpl.java:769)
	at com.intellij.psi.SingleRootFileViewProvider.findElementAt(SingleRootFileViewProvider.java:439)
	at com.intellij.psi.SingleRootFileViewProvider.findElementAt(SingleRootFileViewProvider.java:425)
	at com.intellij.psi.impl.source.PsiFileImpl.findElementAt(PsiFileImpl.java:611)
	at com.intellij.ide.projectView.impl.ProjectViewImpl$MyAutoScrollFromSourceHandler$MySelectInContext.b(ProjectViewImpl.java:1762)
	at com.intellij.ide.projectView.impl.ProjectViewImpl$MyAutoScrollFromSourceHandler$MySelectInContext.getSelectorInFile(ProjectViewImpl.java:1778)
	at com.intellij.ide.impl.SelectInTargetPsiWrapper.selectIn(SelectInTargetPsiWrapper.java:83)
	at com.intellij.ide.projectView.impl.ProjectViewImpl$MyAutoScrollFromSourceHandler.a(ProjectViewImpl.java:1704)
	at com.intellij.ide.projectView.impl.ProjectViewImpl$MyAutoScrollFromSourceHandler.b(ProjectViewImpl.java:1696)
	at com.intellij.ide.projectView.impl.ProjectViewImpl$MyAutoScrollFromSourceHandler.scrollFromSource(ProjectViewImpl.java:1667)
	at com.intellij.ide.projectView.impl.ProjectViewImpl.d(ProjectViewImpl.java:618)
	at com.intellij.ide.projectView.impl.ProjectViewImpl.setupImpl(ProjectViewImpl.java:582)
	at com.intellij.ide.projectView.impl.ProjectViewImpl.setupImpl(ProjectViewImpl.java:543)
	at com.intellij.ide.projectView.impl.ProjectViewToolWindowFactory.createToolWindowContent(ProjectViewToolWindowFactory.java:34)
	at com.intellij.openapi.wm.impl.ToolWindowImpl.ensureContentInitialized(ToolWindowImpl.java:510)
	at com.intellij.openapi.wm.impl.ToolWindowManagerImpl$11.run(ToolWindowManagerImpl.java:520)
	at com.intellij.openapi.application.impl.LaterInvocator$FlushQueue.run(LaterInvocator.java:332)
	at java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:311)
	at java.awt.EventQueue.dispatchEventImpl(EventQueue.java:749)
	at java.awt.EventQueue.access$500(EventQueue.java:97)
	at java.awt.EventQueue$3.run(EventQueue.java:702)
	at java.awt.EventQueue$3.run(EventQueue.java:696)
	at java.security.AccessController.doPrivileged(Native Method)
	at java.security.ProtectionDomain$1.doIntersectionPrivilege(ProtectionDomain.java:75)
	at java.awt.EventQueue.dispatchEvent(EventQueue.java:719)
	at com.intellij.ide.IdeEventQueue.e(IdeEventQueue.java:734)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:569)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.java:382)
	at java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:201)
	at java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:116)
	at java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:105)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:101)
	at java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:93)
	at java.awt.EventDispatchThread.run(EventDispatchThread.java:82)

```

**Comment by KronicDeth** [comment]
* Which IDE
* Which version of the IDE
* Which version of JDK
* Which Operating System

**Comment by KronicDeth** [comment]
Duplicate of #391, but keeping separate until @anehing and @RatanPaul confirm they are using the same install combination.

**Comment by KronicDeth** [comment]
Opened a [question](https://intellij-support.jetbrains.com/hc/en-us/community/posts/207852029-Some-users-of-my-plugin-get-java-lang-NoSuchMethodError-com-intellij-lang-parser-GeneratedParserUtilBase-enter-section-) on the OpenAPI forum to see if JetBrains has any guidance.

**Comment by KronicDeth** [comment]
Fixed by #406
